cmd/fluent-watcher/fluentd/main.go

```diff
@@ -273,12 +273,6 @@ func reloadOrStop() {
 
 	_ = level.Info(logger).Log("msg", "Gracefully reload Fluentd config error", "error", err)
 
-	err = cmd.Process.Signal(syscall.SIGTERM)
-	if err == nil {
```

Comment:
This part is not suggested to remove since this signal will implement the usr2 signal once it failed.

Reply:
Maybe it should be retried after a failed reload instead of just killing the process, which may result in data loss.

Reply:
Agree.

Reply:
@wanjunlei looks like SIGTERM will stop the daemon gracefully. 
The only difference between SIGTERM and SIGUSR2 is SIGUSR2 will recreate the data pipeline:

![image](https://user-images.githubusercontent.com/18525465/167254259-6ff99927-905c-47ce-97bc-b9191e6f5539.png)

Reply:
So maybe we can keep the original SIGTERM code @wanjunlei @wenchajun @zhu733756

Reply:
I think it's good to keep it as it is